### PR TITLE
Adding timer format

### PR DESCRIPTION
**Pull request by EstatoDeviato**

### CODE REVIEW

File: src/courtroom.cpp [hunk 1]
```diff
@@ -6760,6 +6761,17 @@ void Courtroom::handle_clock(QString time)
   ui_rp_clock->show();
 }
 
+void Courtroom::format_clock(QStringList time_format) {
```
Comment (Crystalwarrior) [inline src/courtroom.cpp:6764]: Why is this function responsible for parsing the arguments? Pass the arguments here and handle the QStringList unpacking in packet_distribution.cpp

File: src/packet_distribution.cpp [hunk 2]
```diff
@@ -698,6 +698,12 @@ void AOApplication::server_packet_received(AOPacket *p_packet)
     else if (type == 3)
       w_courtroom->set_clock_visibility(id, false);
   }
+  else if (header == "TF") {
+    if (courtroom_constructed && f_contents.size() > 1) {
+      QStringList time_format = f_contents;
```
Comment (Crystalwarrior) [inline src/packet_distribution.cpp:703]: Break this up into arguments and pass it to format_clock instead of doing it in format_clock

**Review by Crystalwarrior — CHANGES_REQUESTED** [review]
Good stuff, just parse the arguments in packet_distribution rather than courtroom

**Comment by EstatoDeviato** [comment]
Done!
Now it is unpacked in packet_distribution file

**Pull request merged**